recipes/couchdb/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "couchdb" %}
+{% set version = "3.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://couchdb.neighbourhood.ie/downloads/{{ version }}/win/apache-couchdb-{{ version }}.msi  # [win]
+    sha256: f7d354d5ede02d2af748f8d3230bd46b9c3e70e7cd6430ddf2095916d260060e  # [win]
+  - url: https://archive.apache.org/dist/couchdb/source/{{ version }}/apache-couchdb-{{ version }}.tar.gz  # [unix]
+    sha256: 69c9fd6f80133557f68a02e92dda72a4fd646d646f429f45bb8329a30f82f20e  # [unix]
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}  # [unix]
+    - {{ compiler('cxx') }}  # [unix]
+    - make  # [unix]
+  host:
+    - libffi  # [unix]
+    - nspr  # [unix]
```

Comment:
There are [static linking warnings](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=507484&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=2819) for libffi and nspr. Are we static linking or overdepending?

Reply:
Overdepending it looks like